op-node/rollup/derive/l1_block_info.go

```diff
@@ -453,6 +453,19 @@ func L1InfoDeposit(rollupCfg *rollup.Config, sysCfg eth.SystemConfig, seqNumber
 	if isEcotoneButNotFirstBlock(rollupCfg, l2Timestamp) {
 		isIsthmusActivated := isIsthmusButNotFirstBlock(rollupCfg, l2Timestamp)
 		l1BlockInfo.BlobBaseFee = block.BlobBaseFee()
+
+		// Apply Cancun blob base fee calculation if this chain needs the L1 Pectra
+		// blob schedule fix (mostly Sepolia OP-Stack chains).
+		if t := rollupCfg.PectraBlobScheduleTime; t != nil && block.Time() < *t {
```

Comment:
We should be clear in the specs about how this HF works. IIUC it is quite different to most hard forks. Usually, specific behaviour is activated at the hard fork time, if it is not nil, and continues until overriden by another hardfork. Here, specific behaviour is activated for all times up to the hard fork time, if it is not nil, and then _deactivated_ at the hard fork time. 

we want to avoid the idea of an `isPectraBlobScheduleTime`.

Reply:
exactly, it's an optional backwards-hardfork :D